cmd/clusterctl/README.md

```diff
@@ -13,8 +13,8 @@ this repository.**
 
 ### Prerequisites
 
-1. Install [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
-2. Install a [driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md) for minikube. For Linux, we recommend kvm2. For MacOS, we recommend VirtualBox.
+1. Install [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or [kind](https://github.com/kubernetes-sigs/kind#installation-and-usage).
```

Comment:
Should we prefer `kind` here and move the minikube references in another optional section?

Reply:
I put `kind` in the first and also mentioned `kind` as preferred, hope it is OK.